src/server.ts

```diff
@@ -0,0 +1,45 @@
+import * as express from 'express';
+import * as path from 'path';
+import * as logger from 'morgan';
+import * as helmet from 'helmet';
+import * as cors from 'cors';
+import * as compression from 'compression';
+import * as cookieParser from 'cookie-parser';
+import * as bodyParser from 'body-parser';
+
+import { init as dbInit } from './db';
+import CityRouter from './routes/cities';
+
+class Server {
+    public app: express.Application;
+
+    constructor() {
+        this.app = express();
+        this.config();
+        this.routes();
+        dbInit();
```

Comment:
Can we remove this dbInit from here ? We don't want it to be called every time :)